### PR TITLE
fix: remove copyright create/delete/edit options from about event men…

### LINKED ISSUE #887: Copyright edit/create items are in about event menu
### Bug Report    

Copyright edit/create items should be on the copyright card instead

**Would you like to work on the issue?**

No

### PR DESCRIPTION
**Pull request by AkshayCHD**
Fixes #887

Changes: create buttons on the copyright textview to create/edit and delete copyrights and removed them from the overflow menu. 

Screenshots for the change:
screenshot_1523640464
screenshot_1523652359

### CODE REVIEW

**Comment by srv-twry** [comment]
@AkshayCHD Please link the issue to the PR

File: app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventFragment.java [hunk 1]
```diff
@@ -73,6 +70,24 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
+
+        binding.detail.actionChangeCopyright.setOnClickListener(view -> {
+            getFragmentManager().beginTransaction()
+                .replace(R.id.fragment, UpdateCopyrightFragment.newInstance(eventId))
+                .commit();
+        });
+
+        binding.detail.actionDeleteCopyright.setOnClickListener(view -> {
+            getPresenter().deleteCopyright(getPresenter().getCopyright().getId());
+        });
```
Comment (srv-twry) [inline app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventFragment.java]: IMO you can add these three click listeners in a method named ```prepareCopyrightButtons```.

File: app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventFragment.java [hunk 2]
```diff
@@ -114,49 +129,15 @@ private void setupRefreshListener() {
...
-            default:
-                // No implementation
-        }
-        return super.onOptionsItemSelected(item);
-    }
-
-    @Override
-    public void onPrepareOptionsMenu(Menu menu) {
-        super.onPrepareOptionsMenu(menu);
+    public void prepareCopyrightButtons() {
```
Comment (srv-twry) [inline app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventFragment.java]: Here you can rename it to something like ```handleVisibility```.

**Comment by AkshayCHD** [comment]
@srv-twry thanks for the suggestions, i have made the necessary changes @iamareebjamal please review

**Comment by iamareebjamal** [comment]
Change the edit and delete icon colors to grey

File: app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventFragment.java [hunk 3]
```diff
@@ -114,49 +114,33 @@ private void setupRefreshListener() {
...
     }
 
-    @Override
-    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
-        inflater.inflate(R.menu.menu_about_event, menu);
-    }
+    private void addCopyrightListeners() {
+        binding.detail.actionCreateCopyright.setOnClickListener(view -> {
+            getFragmentManager().beginTransaction()
+                .replace(R.id.fragment, CreateCopyrightFragment.newInstance())
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventFragment.java]: Please check what this was before your commit

File: app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventFragment.java [hunk 4]
```diff
@@ -114,49 +114,33 @@ private void setupRefreshListener() {
...
-            case R.id.action_delete_copyright:
-                getPresenter().deleteCopyright(getPresenter().getCopyright().getId());
-                break;
-            default:
-                // No implementation
-        }
-        return super.onOptionsItemSelected(item);
+        binding.detail.actionChangeCopyright.setOnClickListener(view -> {
+            getFragmentManager().beginTransaction()
+                .replace(R.id.fragment, UpdateCopyrightFragment.newInstance(eventId))
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventFragment.java]: Please check what this was before your commit

**Comment by AkshayCHD** [comment]
@iamareebjamal please review

**Pull request merged**